index.html

```diff
@@ -1,16 +1,410 @@
+        <p class="display-4 d-none d-sm-block">Get You Cake Groove On</p>
+        <p class="lead">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Natus,
+          eveniet.
+        </p>
+        <p class="lead">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit.
+        </p>
+        <a href="#" class="btn btn-lg btn-primary">Visit Cake Store</a>
+        <a href="#" class="btn btn-lg btn-primary">Create Your Own</a>
+      </div>
+    </section>
+    <!-- End of Jumbotron-->
+    <div class="container">
+      <section id="about">
+        <h2 class="display-4 text-center">All about Cake R Us</h2>
+        <p>
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatibus
+          delectus magni et similique quos accusantium nisi ullam inventore nemo
+          non blanditiis aperiam eligendi, iusto magnam ducimus nihil ea,
+          consectetur odit facilis suscipit minus qui reiciendis nesciunt.
+          Facilis assumenda odio similique?
+        </p>
+        <p>
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sed, neque
+          nostrum. Itaque officia minus repudiandae, officiis soluta ducimus.
+          Libero, officia.
+        </p>
+      </section>
+      <section id="promo">
+        <h2 class="display-4 text-center">
+          What We Do
+        </h2>
+        <p class="text-center">Watch our Cakes giving Hapiness</p>
+        <div class="embed-responsive embed-responsive-16by9">
+          <iframe
+            src="https://player.vimeo.com/video/70217231?color=ff7dba"
+            width="640"
+            height="560"
+            frameborder="0"
+            allow="autoplay; fullscreen"
+            allowfullscreen
+          ></iframe>
+          <p></p>
+        </div>
+      </section>
+      <section id="showcase">
+        <h2 class="display-4 text-center">
+          See Our Cakes
+        </h2>
+        <p class="text-center">Cakes,Cakes and More Cakes</p>
+        <div id="carousel" class="carousel slide" data-ride="carousel">
+          <ul class="carousel-indicators">
+            <li data-target="#carousel" data-slide-to="0" class="active"></li>
+            <li data-target="#carousel" data-slide-to="1"></li>
+            <li data-target="#carousel" data-slide-to="2"></li>
+            <li data-target="#carousel" data-slide-to="3"></li>
+          </ul>
+          <div class="carousel-inner" role="listbox">
+            <div class="carousel-item active">
+              <img
+                src="/images/slide5.jpg"
+                alt=""
+                hei
+                class="d-block img-fluid"
+              />
+              <div class="carousel-caption">
+                <h3 class="display-4">Cake</h3>
+                <p>Cake, glorious cake</p>
+              </div>
+            </div>
+            <div class="carousel-item">
+              <img src="/images/slide6.jpg" alt="" class="d-block img-fluid" />
```

Comment:
There's no alt-tags on these images. Which means a user with a screenreader would have a problem navigating this section.